web/src/main/java/org/openmrs/module/web/WebModuleUtil.java

```diff
@@ -126,7 +126,7 @@ public static boolean startModule(Module mod, ServletContext servletContext, boo
 				realPath = System.getProperty("user.dir");
 			}
 			
-			File webInf = new File(realPath + "/WEB-INF".replace("/", File.separator));
+			File webInf = new File(realPath, "WEB-INF".replace("/", File.separator));
```

Comment:
@MalshaL you need to remove the string replace call, see that before the "/" was replaced with the File.separator to make it cross-platform. thats not needed when letting File construct the path.